RevenueCatUI/Templates/MultiPackageTemplate.swift

```diff
@@ -142,7 +142,7 @@ private struct MultiPackageTemplateContent: View {
                         : .gray
                     )
 
-                Text(package.content.productName)
+                Text(self.localization(for: package.content).offerName ?? package.content.productName)
```

Comment:
`offerName` isn't required (since not all templates will have it), so I'm making it default to the product name which is always available /cc @guido732 @lburdock